### ISSUE TITLE
"Event High Score" not updating for subsequent matches

### ISSUE THREAD

**Issue by Cannongeorge** [FIRST-Tech-Challenge/scorekeeper#809]
Played a league meet today.  In qualification round #3, audience and pit displays correctly flagged a team as having set a new "Event High Score."  Yet, twice more in the tournament we set subsequent even higher scores in qualification rounds #13 and #14; neither of these rounds flagged the winning team as setting a new "Event High Score."   

I set a 1-hour lunch break in the middle of this, don't know if that broke the chain of keeping track of high scores?  

Ideally, the audience and pit displays would've flagged the subsequent even higher match scores for the teams during this event?  

Running Win 11, Chrome, First Tech Live Challenge Scoring System v6.4.1.

**Comment by adwiii** [comment]
Can you provide more detail about which event this occurred at so we can investigate?

**Comment by Cannongeorge** [comment]
Yes.  The event was held today (12/14/2024); League V and I've uploaded the
archive file to this email.
Let me know if you need any more data.

**Comment by adwiii** [comment]
Watching back the live stream link I found [here](https://www.youtube.com/live/3Jd__RC7ly0), it seems to be working correctly on the streaming display. I pulled (copied below) matches 2, 3, 13, and 14, and all show the high score banner for the winning alliance. Were there other displays in the venue that did not match this behavior? If so, can you elaborate on the situation?

![image](https://github.com/user-attachments/assets/4109bf52-bf4c-4d5c-a37e-fb08057f9b02)
![image](https://github.com/user-attachments/assets/8f240e79-79ea-4031-9970-e05b9eb486ec)
![image](https://github.com/user-attachments/assets/824ee135-3836-45f0-bd00-8547e662fafa)
![image](https://github.com/user-attachments/assets/0e1a2d2e-f652-4a37-af17-446fa2b98a86)

**Comment by Cannongeorge** [comment]
I believe I have figured this out, and the problem is strictly on my end.

It was the “Audience” display that was not working. (This included a
separate computer, and commercial AMX wall screen set up).  At some point
during the match, the display resolution settings for this display were
changed.  As it worked out, it cut off a small horizontal slice of the
uppermost portion of the display which included the “Qualification XX of
YY” identifier and “Event High Score” banner.  I should’ve caught this but
did not notice at the time.


George
